### ISSUE TITLE
[Enhancement] Add docs or functionality for progress circle label

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#3948]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
We should figure out if we can add meaningful labels to progress circles.

## Describe the solution you'd like
Ideally, I think this should be done as a visual label by default, as it is useful for any user.

And as a fallback, we should document how an aria-label and possibly aria-labelledby can be added in those cases where it is not meaningful to have a visual label.

## Have you considered any alternatives?
A current workaround is for consumers to always add an aria-label to any progress circle.

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by henrikvoetmand** [comment]
**Design research**
We have 2 common use scenarios for Progress circle:

**Indicator for a remaining value**

<img width="361" alt="Image" src="https://github.com/user-attachments/assets/b0be9290-775b-4165-a27b-028cf7b34d9d" />

In this case the Card title would be the label or part of the label for the Progress circle.

**Indicator for a flow progress**

![Image](https://github.com/user-attachments/assets/cd669221-e86a-42d4-9fba-67d0d35f1108)
In this case the Modal title would be the label or part of the label for the Progress circle.

It is hard to design a suitable visual label for the progress circle that adds further value and not duplicating the parent container label/title.

If a progress circle were used alone it would require a label, but would this label not come from some kind og section title or other descriptive text?

**Investigation regarding other Design Systems:**
It seems to be the trend that: 
Circular progress has no visual label.
Progress bars (linear) has a visual label

https://carbondesignsystem.com/components/progress-bar/usage/
https://atlassian.design/components/progress-bar/examples
https://v2.chakra-ui.com/docs/components/circular-progress
https://mui.com/material-ui/react-progress/
https://spectrum.adobe.com/page/progress-circle/
https://eui.elastic.co/docs/components/display/progress/
